TAA/TAA.md

```diff
@@ -0,0 +1,141 @@
+
+FOR GOOD AND VALUABLE CONSIDERATION, THE SUFFICIENCY OF WHICH IS HEREBY ACKNOWLEDGED, THE PARTIES AGREE AS FOLLOWS:
+
+## **1)Definitions**
+* **a.** "**Data Protection Laws**" means the GDPR any other data protection and privacy laws, regulations, and regulatory requirements applicable to a party under this Agreement.
+* **b.** "**GDPR**" means the General Data Protection Regulation (EU) 2016/679 on the protection of natural persons with regard to the Processing of personal data and on the free movement of such data, and repealing Directive 95/46/EC, and any amendment or replacement to it.
+* **c.** "**Impermissible Personal Data**" means the Personal Data that Transaction Author writes to the Sovrin Ledger Networks that is not Permissible Personal Data.
+* **d.** "**Personal Data Transactions**" has the meaning set forth in Section 3 below.
+* **e.** "**Permissible Personal Data**" means Personal Data that Transaction Author writes to the Sovrin Ledger Networks that is permitted under this Agreement and the Sovrin
+Governance Framework (including the ​[Sovrin Ledger Access Policies​](https://sovrin.org/wp-content/uploads/Sovrin-Ledger-Access-Policies-V1.pdf)).
+* **f.** "**Personal Data**" means information that relates, directly or indirectly, to a data subject, including without limitation, names, email addresses, postal addresses, identification numbers, location data, online identifiers or one or more factors specific to the physical, physiological, genetic, mental, economic, cultural or social identity of the data subject.
+* **g.** "**Process**" or "**Processing**"​ means any operation or set of operations which is
+performed on Transactions data, whether or not by automated means, such as the access, collection, use, storage, disclosure, dissemination, combination, recording, organization, structuring, adaption, alteration, copying, transfer, retrieval, consultation, disposal, restriction, erasure and/or destruction of Transactions data.
+* **h.** "**Sovrin Governance Framework**" means the Sovrin Foundation’s governance policies and rules available at [​https://sovrin.org/governance-framework/](https://sovrin.org/governance-framework/) or any successor website.
+
+Unless otherwise defined above, all capitalized terms used in this Agreement shall have the meanings given to them in this Agreement or in the ​Sovrin Governance Framework and corresponding ​Sovrin Glossary​. The Sovrin Governance Framework and Sovrin Glossary is incorporated into this Agreement by reference only for purposes of use of such defined terms.
+
+## **2)Permission to Write to the Sovrin Ledger Networks**
+* a. The Sovrin Foundation hereby grants to Transaction Author a non-exclusive, non-assignable, non-sublicensable, royalty free, revocable license to write to and use the Sovrin Ledger Networks in accordance with this Agreement and the Sovrin Governance Framework.
```

Comment:
Making the `a.`, `b.`, etc. bold as in the previous section or removing the bold from the previous section would be good for consistency

Reply:
I agree.

Reply:
nevermind we will remove the bolding.